become-a-karakuni.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: article
+title: You@Karakun
+order: '19'
+headerText: 
+permalink: /become-a-karakuni/
```

Comment:
Can we please either stick with Karakunie or change it to "Karakun". I think the -ie belittlement is not helpful for our representation.

Reply:
makes sense ;)